datashader/tests/test_dask.py

```diff
@@ -42,12 +43,24 @@ def _dask_expr():
 def _dask_cudf():
     import dask_cudf
     _dask = dd.from_pandas(_pandas(), npartitions=2)
-    return dask_cudf.from_dask_dataframe(_dask)
+    if Version(dask_cudf.__version__) >= Version("24.06"):
+        return _dask.to_backend("cudf")
+    else:
+        return dask_cudf.from_dask_dataframe(_dask)
+
+@dask_switcher(query=True, extras=["dask_cudf"])
+def _dask_expr_cudf():
+    import dask_cudf
+    if Version(dask_cudf.__version__) >= Version("24.06"):
+        pytest.skip("dask-expression first requires dask-cudf 24.06")
```

Comment:
Not sure what "first requires means. Maybe:
```suggestion
        pytest.skip("dask-expression requires dask-cudf 24.06 or later")
```